repository.eselect.5.in

```diff
@@ -0,0 +1,169 @@
+.\" -*- coding: utf-8 -*-
+.\" Copyright 2019 Till Schäfer
+.\" Distributed under the terms of the 2-clause BSD license
+.\"
+.TH repository.eselect 5 "31 January 2019" "Gentoo Linux" "eselect"
+.SH NAME
+\fBrepository.eselect\fP - Manage repos.conf via eselect
+.SH SYNOPSIS
+.nf
+.fam C
+\fBeselect\fP \fBrepository\fP [\fBhelp\fP|\fBusage\fP|\fBversion\fP]
+\fBeselect\fP \fBrepository\fP \fBadd\fP \fIname\fP \fIsync-type\fP \fIsync-uri\fP
+\fBeselect\fP \fBrepository\fP \fBdisable\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+\fBeselect\fP \fBrepository\fP \fBenable\fP (\fIname\fP|\fIindex\fP)
+\fBeselect\fP \fBrepository\fP \fBlist\fP [\fB-i\fP]
+\fBeselect\fP \fBrepository\fP \fBremove\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+
+.SH DESCRIPTION
+\fBrepository.eselect\fP is an eselect module for manipulating /etc/portage/repos.conf entries to third party repositories (also known as overlays). This supersedes the need for app-admin/layman to list, configure, and handle synchronizations of alternate repositories except for those which the package manager does not (eg. mercurial, bazaar and g-sorcery in portage).
+.SH ACTION: ADD
+Add a custom repository
+.PP
+\fBeselect\fP \fBrepository\fP \fBadd\fP \fIname\fP \fIsync-type\fP \fIsync-uri\fP
+.PP
+Parameters:
+.TP
+.B
+\fIname\fP
+Name of the repository to add
+.TP
+.B
+\fIsync-type\fP
+Syncer to use (e.g. git, rsync, svn\.\.\.)
+.TP
+.B
+\fIsync-uri\fP
+Sync URI
+.SH ACTION: DISABLE
+Disable specified repositories without removing their contents
+.PP
+\fBeselect\fP \fBrepository\fP \fBdisable\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+.PP
+Parameters:
+.TP
+.B
+\fB-f\fP
+Force potentially dangerous removals. Required for unregistered repositories and those without sync attributes.
```

Comment:
You could explain that dangerous = repositories that can't be readded via enable.